client/src/components/Tag.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import { ReactNode, useState } from "react";
+
+function Tag( { children }: { children: ReactNode }) {
+  const [selected, setSelected] = useState(false);
```

Comment:
For the props, we're probably gonna pass in a useState into this such that it can update the list of things pressed back in the form.